### PR TITLE
Line follower readings

### PR DESCRIPTION
**Pull request by aninternetof**
Adds support for the line sensors.
* Wires values through Redux `sensor` group
* Adds NumericSensorReading to MissionControl UI
* Adds new blockly block to read value

It was essentially copy-paste from the light sensor.

### CODE REVIEW

**Comment by aninternetof** [comment]
Not sure what's up with all the merge commits.

**Comment by cabarnes** [comment]
I'm assuming because your local `alpha` branch was behind

File: src/components/Workspace.js [hunk 1]
```diff
@@ -327,7 +337,9 @@ class Workspace extends Component {
       ),
     );
 
-    this.updateSensorStateCache(sensor.left, sensor.right);
+    this.updateSensorStateCache(sensor.left, sensor.right,
+      sensor.leftLineSensorReading, sensor.rightLightSensorReading,
```
Comment (cabarnes) [inline src/components/Workspace.js:341]: Should be light

Comment (aninternetof) [reply src/components/Workspace.js:341]: Good catch. Fixed.

**Pull request merged**